src/pyrokinetics/equilibrium/geqdsk.py

```diff
@@ -197,7 +198,7 @@ def verify_file_type(self, filename: PathLike) -> None:
             ) from exc
         # Check that the correct variables exist
         var_names = ["nx", "ny", "simagx", "sibdry", "rmagx", "zmagx"]
-        if not np.all(np.isin(var_names, list(data.keys()))):
+        if not np.all(np.isin(var_names, list(asdict(data)))):
```

Comment:
Actually, I don't think this check is needed at all -- we're guaranteed to have all of these keys. So just delete L199-204?

Reply:
Good point, I'm not sure why this check was ever needed in the first place.